src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -55,7 +55,7 @@ fun main() {
 
             module {
                 auth(wellKnown, listOf(env.loginserviceClientId))
-                serverModule(applicationState)
+                serverModule(applicationState, database, env)
```

Comment:
Ser litt rart ut å sende inn "database" som argument her siden den initialiseres av databaseModule, men det funker kanskje likevel?

Reply:
Jeg er enig. Tror ikke det er nødvendig. iom at den importeres for å sendes nedover.